test/module/irohad/ametsuchi/postgres_query_executor_test.cpp

```diff
@@ -375,6 +396,11 @@ namespace iroha {
                         true)));
       }
 
+      // TODO mboldyrev 05.12.2018 IR-57 unify the common constants.
+      // Some of them, like account2, are defined multiple times, but seem
```

Comment:
You can place the description about constants into the task's description

Reply:
As I read the [IR-57](https://jira.hyperledger.org/browse/IR-57) description, I realized the changes it requests are way more abstract than the constants refactoring. It seems the whole classes will be rewritten. So I decided not to distract that change's objective with such small matters.